tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py

```diff
@@ -0,0 +1,138 @@
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='RACE_WHITE', result_function=validators.isInLimits(1, 2),
+                  ),
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='MARITAL_STATUS', result_function=validators.isInLimits(1, 5),
+                  ),
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 2),
+                        result_field='PARENT_MINOR_CHILD', result_function=validators.isInLimits(1, 3),
+                  ),
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.isInLimits(1, 3),
+                        result_field='EDUCATION_LEVEL', result_function=validators.or_validators(
+                            validators.isInStringRange(1, 16), validators.isInStringRange(98, 99)),
+                  ),
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                        result_field='CITIZENSHIP_STATUS', result_function=validators.isInLimits(1, 3),
+                  ),
+              validators.if_then_validator(
+                        condition_field='DATE_OF_BIRTH', condition_function=validators.olderThan(18),
+                        result_field='REC_OASDI_INSURANCE', result_function=validators.isInLimits(1, 2),
+                  ),
+              validators.if_then_validator(
+                        condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                        result_field='REC_FEDERAL_DISABILITY', result_function=validators.isInLimits(1, 2),
+                  ),
+          ],
+          fields=[
+              Field(item="0", name='RecordType', type='string', startIndex=0, endIndex=2,
+                    required=True, validators=[]),
+              Field(item="3", name='RPT_MONTH_YEAR', type='number', startIndex=2, endIndex=8,
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
+                                               validators.dateMonthIsValid(),]),
+              Field(item="5", name='CASE_NUMBER', type='string', startIndex=8, endIndex=19,
+                    required=True, validators=[validators.isAlphaNumeric()]),
+              Field(item="13", name='FAMILY_AFFILIATION', type='number', startIndex=19, endIndex=20,
+                    required=True, validators=[]),
+              Field(item="14", name='DATE_OF_BIRTH', type='string', startIndex=20, endIndex=28,
+                    required=True, validators=[validators.dateYearIsLargerThan(1998),
```

Comment:
should date of birth be limited for the case record?

Reply:
Good question. I used this validator since it is what Mo used in TANF section 1 validators for the date of birth. Just some copy and paste. Open to updating!